README.md

```diff
@@ -2,6 +2,37 @@
 
 The Wikipedia Library Card Platform app, available at http://wikipedialibrary.wmflabs.org.
 
+## Quick Installation for Developers
+
+- Get Docker and Docker Compose
+- Fire up an empty TWLight instance `docker-compose build && docker-compose up`
+- Run migrations `docker-compose exec twlight /app/bin/virtualenv_migrate.sh`
```

Comment:
Is there a reason we wouldn't want to always run migrations in the Dockerfile?

Reply:
- we don't have a database at build time (when the dockerfile is in play), but on the servers, I am automatically running the migration upon having the image come up.
- there are situations in which you really don't want automatic migration generation in dev, like when you are working on code that requires a custom migration. This occasionally bit me in the vagrant environment, and I'd have to run through some hoops to avoid the migrations.

Having said that, we should probably have feature parity with the vagrant environment in terms of what you get when you bring the environment up. I'm kind of an edge case.

Reply:
Gotcha - that totally makes sense, and one line isn't really an issue, I was just curious.